bots/tester/src/test/java/org/openjdk/skara/bots/tester/InMemoryHostedRepository.java

```diff
@@ -234,6 +234,11 @@ public URI webUrl(Tag tag) {
         return null;
     }
 
+    @Override
+    public URI url() {
+        return null;
```

Comment:
It probably doesn't matter much, but seems like this should return `url` just like `authenticatedUrl()` does.